src/playground/with-layout/chat/chat-custom-message.component.html

```diff
@@ -0,0 +1,34 @@
+<nb-chat title="Nebular Conversational UI" size="large">
+    <nb-chat-message *ngFor="let msg of messages"
+        [type]="msg.type"
+        [message]="msg.text"
+        [reply]="msg.reply"
+        [sender]="msg.user.name"
+        [date]="msg.date"
+        [files]="msg.files"
+        [quote]="msg.quote"
+        [latitude]="msg.latitude"
+        [longitude]="msg.longitude"
+        [avatar]="msg.user.avatar"
+        [customMessageData]="msg.customMessageData">
+
+        <div *nbCustomMessage="'link'; let data">
+            <a [href]="data.href">{{ data.label }}</a>
+        </div>
+        <div class="button-wrapper" *nbCustomMessage="'button'; let data">
+            <button class="custom-button">{{ data }}</button></div>
+
+        <div *nbCustomMessage="'img'">
+            <picture>
+                <img src="https://i.gifer.com/no.gif">
+            </picture>
+        </div>
```

Comment:
Let's put some video instead.

Reply:
Will stay with picture. Video may be in future.